servers/streamlare.py

```diff
@@ -23,23 +23,13 @@ def test_video_exists(page_url):
 def get_video_url(page_url, premium=False, user="", password="", video_password=""):
     global data, response
     logger.info("(page_url='%s')" % page_url)
-    # from core.support import dbg
-    # dbg()
+
     video_urls = []
     id = scrapertools.find_single_match(page_url, '/e/(\w+)')
     post = {"id": id}
-    token = scrapertools.find_single_match(data, '<meta name="csrf-token" content="([^"]+)')
-    data = httptools.downloadpage("https://streamlare.com/api/video/stream/get",
-                                  post=post, headers={'X-CSRF-TOKEN': token, 'X-Requested-With': 'XMLHttpRequest',
-                                                      'X-XSRF-TOKEN': httptools.urlparse.unquote(response.cookies.get('XSRF-TOKEN'))}).json
-    # src = data["result"]["Original"]["src"]
-    # media_url = ''.join([chr(51 ^ c) for c in base64.b64decode(src)])
-    media_url = data["result"]["file"]
-    video_urls.append(["MP4", media_url])
-    return video_urls
-
-
-def get_filename(page_url):
```

Comment:
Questa funzione lasciala, serve in cinemalibero per riconoscere il numero episodio avendo solo il link (da verificare se funziona ancora)

Reply:
Ok ho aggiunto la def, riesci a vederla?